### PR TITLE
Added capital invesment banner and urls from directory constants to eu exit and set up guide

### PR DESCRIPTION
**Pull request by jessgilbert**
Eu Exit and Set up guide goes to correct international urls and the capital investment banner

### CODE REVIEW

File: core/sass/pages/landing-page.scss [hunk 1]
```diff
@@ -205,3 +205,40 @@
     }
   }
 }
+
+.center_vertical{
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.case-study {
```
Comment (MichaelWalker) [inline core/sass/pages/landing-page.scss]: I'm not sure that this class name is accurate any more
Did this come directly from the case study on the sector pages? Did we need to do any changes in the end?

If we didn't need to change anything then perhaps we should raise a new PR to port it over to directory-components?

File: core/sass/pages/landing-page.scss [hunk 2]
```diff
@@ -205,3 +205,40 @@
     }
   }
 }
+
+.center_vertical{
```
Comment (MichaelWalker) [inline core/sass/pages/landing-page.scss]: (Question for @eadpearce I think)
Do we already have a common way of centring things that we could 'just use'?

File: core/sass/pages/landing-page.scss [hunk 3]
```diff
@@ -205,3 +205,40 @@
...
+      .button {
+        width: auto;
+      }
+    }
+
+    .button::after {
+      content: '→';
+      margin-left: 1em;
+    }
+  }
```
Comment (Kenneth-Siu) [inline core/sass/pages/landing-page.scss]: Is the css namespaced? Does it not matter for invest-ui? (Not worked on invest-ui before)

Comment (Kenneth-Siu) [reply core/sass/pages/landing-page.scss]: If that's meaningless, happy to explain in person.

File: conf/settings.py [hunk 4]
```diff
@@ -88,6 +88,9 @@
                 'django.template.context_processors.i18n',
                 'core.context_processors.untranslated_url',
                 'core.context_processors.footer_contact_us_link',
+                'core.context_processors.how_to_do_business_link',
+                'core.context_processors.set_up_in_uk_international_link',
+                'core.context_processors.capital_invest_landing_page_link',
```
Comment (richtier) [inline conf/settings.py]: the values provided by the context processor is only used by `landing_page.html`. can you move them to the view's `get_context_data?`

File: core/templates/core/landing_page.html [hunk 5]
```diff
@@ -38,28 +38,41 @@ <h2 class="heading-large">{{ page.benefits_section_title }}</h2>
...
 					{% include 'directory_components/image_with_caption.html' with image_url=page.benefits_section_img.url main_caption=page.benefits_section_img_caption  %}
 				</div>
 			{%  endif %}
 		</div>
   </div>
 </section>
 
+<section id="sector-case-study-section"
+class="case-study"
+style="background-image: url({{ page.capital_invest_section_background_image.url }}) " >
```
Comment (richtier) [inline core/templates/core/landing_page.html]: I kind of like putting attributes on new lines (especially when working with react) but it's inconsistent with the prevailing approach in this codebase. please put use a single line so we don't have competing ways of working :)

**Pull request merged**